source/includes/_tipser-elements-analytics.md

```diff
@@ -74,6 +74,34 @@ The code above assumes that you use <a href="https://developers.google.com/analy
 
 For the instructions how to setup analytics.js script on your site and connect it to your GA account, refer to the <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs" target="_blank">official documentation</a>.
 
+## Typical use case: Facebook Pixel
+
+Facebook Pixel is also an option to tunnel Tipser Analytics events, here's a snippet:
+
+```javascript
+document.addEventListener('tw-track', function(e) {
+    // fbq() function coming from Facebook Pixel analytics script
+    fbq('trackCustom', e.detail.action, {
+        content_name: e.detail.target, 
+        content_category: e.detail.description,
+        content_type: e.detail.action,
+    });
+});
+```
+Example above doesn't translate tipser events to pixel events, you will have to make corelation in pixel dashboard or you can use function that translates those for you. 
+
+1. Go <a href="https://gist.github.com/sirpeas/114bba643e920b0f53dff9487670f509" target="_blank">here</a> and copy the script.
+2. Paste it above addEventListener for tw-track event.
+3. Use it like on belowed example
```

Comment:
This sentence is not grammatical, possible replacements:
- Use it as demonstrated below
- Use it the same way as in the example below (by @AgaK-O )
- Use it in the following way